src/styles/utills/mixin.scss

```diff
@@ -0,0 +1,17 @@
+@mixin fourColumn {
+  @media (min-width: 1024px) {
+    @content;
+  }
+}
+
+@mixin threeColumn {
+  @media (min-width: 768px) {
+    @content;
+  }
+}
+
+@mixin twoColumn {
+  @media (min-width: 488px) {
+    @content;
+  }
+}
```

Comment:
It's not about columns, it's about styles that apply to different devices. So it's better to name such as "onTablet", "onMobile", etc.